BondInventoryControl/.gitignore

```
@@ -0,0 +1,5 @@
+/.setting/
+/bin/
+.classpath
+.project
+org.eclipse.jdt.core.prefs
```

ファイルの最後の行は改行で終了するようにしましょう。
[ソースファイルの末尾に }](http://www.kijineko.co.jp/tech/superstitions/last-letter-of-file-is-brace.html)
↑C言語の記事だけど、普通のファイルもJavaファイルも同様。
```suggestion
org.eclipse.jdt.core.prefs

```